src/datadog/remote_config.cpp

```diff
@@ -43,11 +44,36 @@ constexpr std::array<uint8_t, sizeof(uint64_t)> capabilities_byte_array(
 }
 
 constexpr std::array<uint8_t, sizeof(uint64_t)> k_apm_capabilities =
-    capabilities_byte_array(APM_TRACING_SAMPLE_RATE);
+    capabilities_byte_array(APM_TRACING_SAMPLE_RATE | APM_TRACING_TAGS);
 
 constexpr StringView k_apm_product = "APM_TRACING";
 constexpr StringView k_apm_product_path_substring = "/APM_TRACING/";
 
+Expected<std::unordered_map<std::string, std::string>> parse_tags(
+    const std::vector<StringView>& list_of_tags) {
```

Comment:
Reconsider whether this is worth sharing in, e.g., `parse_util`. Duplicated it here looks fine to me, though.

Reply:
That's something I will reconsider with active configuration :)